packages/@react-spectrum/table/src/DragPreview.tsx

```diff
@@ -11,73 +11,29 @@
  */
 
 import {classNames} from '@react-spectrum/utils';
-import {Direction, Node} from '@react-types/shared';
 import {Flex} from '@react-spectrum/layout';
-import {GridNode} from '@react-types/grid';
 import React from 'react';
 import type {SpectrumTableProps} from './TableView';
 import styles from '@adobe/spectrum-css-temp/components/table/vars.css';
 import stylesOverrides from './table.css';
-import type {TableLayout} from '@react-stately/layout';
 
 interface DragPreviewProps<T> {
-  item:  Node<T>,
-  children: Iterable<Node<T>>,
+  itemText: string,
   itemCount: number,
   density: SpectrumTableProps<T>['density'],
-  layout: TableLayout<T>,
-  direction: Direction,
-  maxWidth: number,
-  rowHeaderColumnKeys: Set<React.Key>
-}
-
-interface TableCellPreviewProps {
-  cell: GridNode<any>
-}
-
-export function TableCellPreview(props: TableCellPreviewProps) {
-  let {cell} = props;
-
-  return (
-    <div
-      className={
-          classNames(
-            styles,
-            'spectrum-Table-cell',
-            classNames(
-              stylesOverrides,
-              'react-spectrum-Table-cell'
-            )
-          )
-        }>
-      <span
-        className={
-            classNames(
-              styles,
-              'spectrum-Table-cellContents'
-            )
-        }>
-        {cell.rendered}
-      </span>
-    </div>
-  );
+  height: number,
+  maxWidth: number
 }
 
 export function DragPreview(props: DragPreviewProps<unknown>) {
   let {
-    item,
-    children,
+    itemText,
     itemCount,
     density,
-    layout,
-    maxWidth,
-    rowHeaderColumnKeys
+    height,
+    maxWidth
```

Comment:
Mega nit: noticed that we are using `UNSAFE_style={{height, maxWidth}}` but we can just apply those as `height` and `maxWidth` and forgo the `UNSAFE_style` right? Also does `density` actually do anything since we get the actual height already?